### PR TITLE
update REQUIRE and run

### PR DESCRIPTION
**Pull request by mforets**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -71,46 +71,45 @@ $ ./install.sh
...
-have their dependencies, this whole process will take between 5-20 minutes in a fresh
-Julia installation.
+have their dependencies, this whole process will take between 5-20 minutes in a
+fresh Julia installation.
 
 ## Running the benchmarks
 
 To run and get the benchmark results in the file `results.md`, execute the
-bash script `run.sh` with the command:
+Julia script `run.julia` with the command:
```
Comment (schillic) [inline README.md:82]: `.jl`?

**Review by schillic — COMMENTED** [review]
EDIT: After I cloned the repository in Julia, it worked. Not sure why...no new package got installed.

**Comment by schillic** [comment]
Should we remove the `run.sh` to avoid confusion?

**Comment by schillic** [comment]
The line `julia --color=yes -e 'Pkg.clone("https://github.com/JuliaReach/ARCH2018_RE.git")'` in `install.sh` confuses me. Why would one clone this package and then clone again?
Ah, the idea is that you only download this file, okay.

**Comment by schillic** [comment]
There are problems ahead: If we change `LazySets`, e.g., in [#472](https://github.com/JuliaReach/LazySets.jl/pull/472), we need to make a new release. Otherwise `Reachability.jl` will use an old version and we are in an inconsistent state.

**Comment by mforets** [comment]
alright, a strategy is:

- make a new minor release of lazysets
- make a release of reachability
- we should `Pin` the correct version numbers .. where,? in `run.jl`?

**Comment by mforets** [comment]
> we should Pin the correct version numbers .. where,? in run.jl?

we can give the version number to be used when we give the installation instructions in the Readme. we could also add CI and pin the version in the travis script.

**Pull request merged**